tv_apps/fling-player/test/unit/connector_test.js

```diff
@@ -0,0 +1,301 @@
+/* global expect, Connector, castingMsgTemplate, MockPresentation */
+'use strict';
+
+requireApp('fling-player/test/unit/casting_message_template.js');
+requireApp('fling-player/test/unit/mock_presentation.js');
+requireApp('fling-player/bower_components/evt/index.js');
+requireApp('fling-player/js/casting_message.js');
+requireApp('fling-player/js/master_debug.js');
+requireApp('fling-player/js/connector.js');
+
+suite('fling-player/Connector', function() {
+
+  var exp, msg, spy, connector, presentation;
+
+  setup(function (done) {
+    var timer;
+    presentation = new MockPresentation();
+    connector = new Connector(presentation);
+    presentation._controller.start();
+    connector.init();
+
+    this.timeout(200);
+    timer = setInterval(() => {
+      if (connector._isInit && connector._isInitConnection) {
+        clearInterval(timer);
+        done();
```

Comment:
Can we find some way to figure out wether the initialization is done, rather than do this kind of polling?